src/Parser.php

```diff
@@ -206,7 +206,7 @@ public function __construct()
         // ? provides for relaxed parsing of messages without trailing parameters properly demarcated
         $trailing = "(?: :?[^$null$crlf]*)";
         $params = "(?P<params>$trailing?|(?:$middle{0,14}$trailing))";
-        $name = "[$letter$number](?:[$letter$number\\-]*[$letter$number])?";
+        $name = "[$letter$number](?:[$letter$number:\/\\-]*[$letter$number])?";
```

Comment:
Correct. @elazar could you add a commit? I don't want to create a pull request just for this line. However we could wait till we got a decision on #34.